Applications/MainApplication/Gui/MaterialEditor.cpp

```diff
@@ -64,7 +67,7 @@ void MaterialEditor::onKsColorChanged( int ) {
     {
         Core::Utils::Color c( ksR->value() / 255.f, ksG->value() / 255.f, ksB->value() / 255.f,
```

Comment:
I saw several ".f" suffix in the code. Should they be "_ra" instead (or just nothing ?)

Reply:
Thanks. Fixed.